back/node_watcher/datamodel.prisma

```diff
@@ -172,4 +172,21 @@ type ReferendumStatus {
   blockNumber: BlockNumber! @relation(name: "ReferendumStatusAtBlockNumber", link: INLINE)
   referendum: Referendum! @relation(name: "ReferendumStatusFromReferendum", link: INLINE)
   status: String!
-}
\ No newline at end of file
+}
+
+type Treasury {
+  id: ID! @id
+  proposer: String!
+  beneficiary: String!
+  value: String!
+  bond: String!
+  proposalId: Int! @unique
```

Comment:
I'd call that `treasuryProposalId` to mirror the `motionProposalId`, and leave `proposalId` to public proposals only.

Reply:
done